templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -297,7 +297,9 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.            compression='uncompressed'
</pre><p>I looked at an issue you investigated a while back, #860 about feather alternatives. It seemed that you concluded that feather files are pretty solid, and they have efficient compression, but they it seems to lack on the actual storage savings side of things. Do you think it may be worth revisiting the compression aspect to see if <code>zstd</code> on the scale of a cohort would be beneficial?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/999/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Definitely worth looking into in the near future.